src/crossval.jl

```diff
@@ -2,7 +2,7 @@
 
 ## cross validation generators
 
-abstract CrossValGenerator
+abstract type CrossValGenerator end
```

Comment:
This is 0.6-only syntax. To support earlier versions, it needs to have an `@compat` in front.